semgrep_output_v1.atd

```diff
@@ -606,14 +603,30 @@ type sca_match_kind = [
 ] < ocaml repr="classic">
 
 type transitive_reachable = {
-  (* TODO: some proof that the user can understand *)
-  explanation: string option;
+  (* The matches we found in 3rd party libraries.
+   * Ideally the location in cli_match are relative to the root of the project
+   * so one can display matches as package@/path/to/finding.py
+   * TODO: include matched code? 3rd party libraries are usually OSS
+   * so maybe ok to store code for once in our DBs? Otherwise The App will
+   * have to redownload and reaccess the package code (or maybe we
+   * could give a github URL?)
+   *)
+  matches: (string (* package uri? *) * cli_match list) list;
```

Comment:
I think `cli_match` includes `found_dependency`, we may not need the string here

Reply:
hmm but when we run semgrep on third party code we remove the SCA part, so we should not get found_dependency.

Reply:
but yes I could use `(found_dependency * cli_match list) list`